docs/migrating-to-velero.md

```diff
@@ -0,0 +1,47 @@
+# Migrating from Ark to Velero
+
+Users of Ark will need to move their installation to the renamed Velero images and Custom Resource Definitions (CRDs). This is a one-time operation.
```

Comment:
Let's make it clear that they must be on v0.10.x for this to work and provide a link to the GH pages site with v0.10 upgrade instructions